File: codingTest/Programmers/3진법 뒤집기/src.py
def solution(n):
    answer = 0
    arr = []
    r = n

    # 3진법으로 바꾸기
    while n != 0:
        r = n % 3
        n = n // 3
        arr.append(r)

    sz = len(arr)-1
    # 10진법으로 표현
    for i, v in enumerate(arr):
        answer += (v*(3**(sz-i)))
    return answer

File: codingTest/Programmers/네트워크/src.cpp
#include <string>
#include <vector>
#include <iostream>
using namespace std;
bool visited[200][200];
void dfs(int x, int y, vector<vector<int>> &computers, int &n){
    visited[x][y] = true;
    visited[y][x] = true;
    
    for(int j = 0; j<n;j++){
        if(!visited[y][j] && computers[y][j] == 1){
            dfs(y, j, computers, n);
        }
    }
}

int solution(int n, vector<vector<int>> computers) {
    int answer = 0;
    for(int i = 0; i < n; i++){
        for(int j = 0; j < n; j++){
            if(computers[i][j] == 1 && !visited[i][j]){
                cout << i << "E" << j ;
                dfs(i, j, computers, n);
                answer++;
            }
        }
    }
    
    return answer;
}
